cmd/wallet-web/src/pages/Dashboard.vue

```diff
@@ -139,42 +161,55 @@ export default {
         return { content: contents[id], flipped: false };
       };
 
-      this.cards = Object.keys(contents).filter(_filter).map(_createCard);
+      this.credentials = Object.keys(contents).filter(_filter).map(_createCard);
 
-      console.log(`showing ${this.cards.length} credentials`);
+      console.debug(`showing ${this.credentials.length} credentials`);
 
-      // Reading the values of the credentials and mapping it to the credential display data schemas
-      this.cjson.forEach((obj) => {
-        let flattened = {};
-        for (let credentialKey in this.cards) {
-          this.flatten(this.cards[credentialKey].content, flattened);
-          for (let credentialContent in flattened) {
-            if (
-              obj.credentialSubject.hasOwnProperty(credentialContent) &&
-              obj.schema === this.credDisplayName(this.cards[credentialKey].content)
-            ) {
-              obj.credentialSubject[credentialContent] = flattened[credentialContent];
-            }
+      Object.entries(this.credentials).forEach(([key]) => {
+        const cType = this.filterType(this.credentials[key].content);
```

Comment:
You looped on object entries and extract key inside loop ? `[key]`?
credential should be function arg here.